src/Phinx/Console/Command/Create.php

```diff
@@ -113,8 +130,54 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ));
         }
 
-        // load the migration template
-        $contents = $this->getMigrationTemplate();
+        // Get the alternative template and static class options, but only allow one of them.
+        $altTemplate = $input->getOption('template');
+        $creationClassName = $input->getOption('class');
+        if (!empty($altTemplate) && !empty($creationClassName)) {
+            throw new \InvalidArgumentException('Cannot use --template and --class');
+        }
+
+        // Verify the alternative template file's existence.
+        if (!empty($altTemplate) && !file_exists($altTemplate)) {
```

Comment:
Using `is_file` is recommended over `file_exists`, because the latter will also return `true` for directories.